docs/apidocs/ansatz_generation.rst

```diff
@@ -1,10 +1,20 @@
-.. _ansatz_generation:
-
-#################
-Ansatz Generation
-#################
+####################################################################
+Ansatz Generation (:mod:`qiskit_addon_aqc_tensor.ansatz_generation`)
+####################################################################
 
 .. automodule:: qiskit_addon_aqc_tensor.ansatz_generation
    :no-members:
    :no-inherited-members:
-   :no-special-members:
\ No newline at end of file
+   :no-special-members:
+
+.. currentmodule:: qiskit_addon_aqc_tensor.ansatz_generation
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+
+    AnsatzBlock
+    OneQubitAnsatzBlock
+    TwoQubitAnsatzBlock
```

Comment:
From above:

> I didn't inline because we show the inherited `Gate` methods, which are long although I think very useful to users

I wonder if these should be marked as "abstract" somehow, e.g. by deriving from both `abc.ABC` and `Gate`.  I haven't looked at the Sphinx output, but all the `Gate` methods are probably not necessary on these three classes because I don't expect any concrete realizations of them.

Not a big deal right now; something to follow up later.